pilot/lib/pilot/tester_manager.rb

```diff
@@ -91,21 +95,35 @@ def find_app(app_filter: nil)
       nil
     end
 
-    def find_or_create_tester(email: nil, first_name: nil, last_name: nil)
-      tester = Spaceship::Tunes::Tester::Internal.find(config[:email])
-      tester ||= Spaceship::Tunes::Tester::External.find(config[:email])
+    def find_or_create_tester(email: nil, first_name: nil, last_name: nil, app: nil)
+      current_user = Spaceship::Members.find(Spaceship::Tunes.client.user)
+      tester = nil
```

Comment:
You don't need to init this with `nil`, if you set the value inside an `if/else` block